src/pages/home/sidebar/SidebarLinks.js

```diff
@@ -142,7 +142,7 @@ class SidebarLinks extends React.Component {
 
         return (
             <View
-                accessibilityElementsHidden={!this.props.isFocused}
+                isFocused={this.props.isFocused}
```

Comment:
Is all this necessary for mocking jest? I can't imagine it is, so it's just strange to see these changes in this PR

Reply:
`accessibilityElementsHidden` was used before both for checking the value in tests and probably to mock the navigation behavior of accessibility (see: https://github.com/react-navigation/react-navigation/blob/9fe34b445fcb86e5666f61e144007d7540f014fa/packages/stack/src/views/Stack/CardContainer.tsx#L242). If we still want to check if this route is focused or not, we need to keep a way of doing it. If you know a better way to do it, it would be great 😅

Reply:
OK, that explains why `accessibilityElementsHidden` is there in the first place, but I guess I am still not quite understanding why it needs to change from `accessibilityElementsHidden` to `isFocused`. We can't keep the original way, or it's not desired for some reason?

Reply:
Hmm `accessibilityElementsHidden` should not be needed anymore, but we can keep it. I just wanted a prop that does not do anything so I didn't destroy any of the behaviors, but since the screen is not focused when this prop is set, it should all work just fine.